### ISSUE TITLE
Unexpected Error when browsing some Games in Store: SteamOS BPM

### ISSUE THREAD

**Issue by Daddoggety** [ValveSoftware/SteamOS#634]
#### Your system information

* Steam client version: April 2nd, 2018
* SteamOS version: 2.148
* Opted into Steam client beta?: No
* Opted into SteamOS beta?: No 
* Have you checked for updates in Settings > System?: Yes
Hardware: i3-8100, MSI Z370M Mortar Mobo, 8Gb system RAM, NVIDIA 1050Ti 4Gb Graphics card

#### Please describe your issue in as much detail as possible:
I selected a game in BPM Steamos Store and expected to see the game details and options to buy as occurs for 95% of games selected in the store.  For some games when I select them I am initially taken to a screen wanting to confirm my date of birth, and, after entering that I receive an **"An error has occurred An unexpected error has occurred while retrieving information from steam"** and no display of the game details page.  From that point on every time I select that game in the store it bypasses the date of birth confirmation screen and goes straight to the error screen until I reboot. 
The following games I know currently cause this issue:
SPEC OPS: The Line
Rise of the Tomb Raider

Unable to purchase these games consequently!!!!!!

#### Steps for reproducing this issue:

1.  Boot Steamos (BPM)
2.  Select Store
3.  Browse games 
4.  Select one of the games mentioned above

**Comment by kisak-valve** [comment]
Hello @Daddoggety, this looks to be an OS-independent issue. Let's track it in the steam-for-linux issue tracker at https://github.com/ValveSoftware/steam-for-linux/issues/5454 for now.

**Comment by Daddoggety** [comment]
Thanks,

 

Some more information:

I switched to the steamos desktop and installed the steam desktop under the desktop user.

I was able to see Rise of the Tomb Raider in desktop mode and even purchase the game(switched back

To steamos and installed the game).

When in desktop steam I switched to BPM and the same issue occurred as occurred with steamos, the

Game information for ROTTR and SPEC OPS: the line gave the error I reported.

 

Of Note: The ROTTR game requires NVIDIA 396 drivers and steamos 2.148 only has NVIDIA 387 drivers,

Perhaps this is why I can’t see these games, because I can’t run them yet in steamos?

 

Cheers, Daddoggety
